tc/core/cpu/cpu_backend.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <glog/logging.h>
+
+#include "tc/core/cpu/cpu_mapping_options.h"
+#include "tc/core/cpu/cpu_mapping_options_cpp_printer.h"
+// TODO: activate when ready
+// #include "tc/core/cpu/cpu_rtc.h"
+#include "tc/core/halide_utils.h"
+#include "tc/core/tensor.h"
+
+namespace tc {
+/**
+ * This is the class that holds a binary object resulting from compilation.
+ * It must be provide a clear function and be callable.
```

Comment:
Drop "be".  Maybe also replace "clear" by "clear()" to clarify that "clear" is the name
of the function rather than a qualification of the function.